src/webpack/util/createStatsFormatter.ts

```diff
@@ -58,10 +58,7 @@ export default function createStatsFormatter(rootPath: string) {
       lines.push('')
     }
 
-    // Make sure we have a valid error message
-    if (err.message && typeof err.message === 'string') {
-      lines.push(formatErrorMessage(err.message))
-    }
```

Comment:
@skjnldsv I wasn't completely sure whether this change is required, so I've backed it out for now (along with some error-related code changes elsewhere in this file).

Reply:
It was just a sanity check on an error I encountered. Where the error was not valid :)